templates/admin/amp-status.php

```diff
@@ -39,36 +39,7 @@
 			</fieldset>
 		<?php else : ?>
 			<div class="inline notice notice-warning notice-alt">
-				<p>
-					<?php
-					$error_messages = array();
```

Comment:
This section is moved to [get_raw_error_messages()](https://github.com/Automattic/amp-wp/pull/1275/files#diff-87f0ef41a80eed0bfff797f932886230R301).